spec/services/mixtape_maker_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe MixtapeMaker do
+  describe '.generate' do
+    it 'makes a correctly formatted mixtape' do
+      artists = create_list(:artist, 3)
+      songs = create_list(:song, 4, artist: artists[0])
+      mixtape = MixtapeMaker.generate(2, artist_ids: artists.collect{ |obj| obj.id})
+      expect(mixtape).to be
+    end
+
+    it 'makes an empty mixtape' do
+      artist = create(:artist)
+
+      mixtape = MixtapeMaker.generate(0, artist_ids: artist)
```

Comment:
Hm, you are passing an `Artist` entity for the `artist_ids`. 

Think you want:

`mixtape = MixtapeMaker.generate(0, artist_ids: artist.id)`